doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2371,6 +2373,19 @@ The issues in this section are related to :ref:`libraries`.
 Binary libraries
 ================
 
+.. rst-class:: v2-6-0
+
+NCSDK-26682: In the Verizon network, the :ref:`liblwm2m_carrier_readme` library fails to complete bootstrap process unless the same device has previously completed a bootstrap
+  This is because one of the required pre-shared keys (PSK) is not generated unless there is a pre-existing one in the modem.
```

Comment:
```suggestion
  This is because one of the required pre-shared keys is not generated unless there is a pre-existing one in the modem.
```
Abbreviation is not required here as the term is in the plural form. Also, confirmed with Uma.